lib/services/validatorsService.ts

```diff
@@ -0,0 +1,21 @@
+
+import {StringTable} from "../models.ts";
+import {MAX_DATASET_SAMPLES} from "../settings.ts";
+
+// Validator for datasetSample
+export const validateDatasetSample = (datasetSample: StringTable): void => {
+    if (datasetSample.length > MAX_DATASET_SAMPLES) {
```

Comment:
Not really sure if this is something we want to validate. The idea is that they are aware that they should only provide a sample not the full dataset

Reply:
@afonsobspinto agree, this constraint make some sense but can also become a blocker. I suggest to remove it